app/controllers/spotlight/catalog_controller.rb

```diff
@@ -179,7 +179,7 @@ def uploaded_resource_params
     end
 
     def custom_field_params
-      current_exhibit.custom_fields.pluck(:field)
+      current_exhibit.custom_fields.where.not(readonly_field: true).pluck(:field)
```

Comment:
Should we make this a scope? Either `current_exhibit.custom_fields.writeable.pluck` or `current_exhibit.writeable_custom_fields`?